nemo/backends/pytorch/actions.py

```diff
@@ -1022,7 +1029,7 @@ def __old_call__(self, force_pt=False, *input, **kwargs):
                 else:
                     return NeuralModule.__call__(self, **kwargs)
 
-            type(module).__call__ = __old_call__
+            type(module).__calll__ = __old_call__
```

Comment:
typo here: __calll__ -> __call__

Reply:
wow, honestly have no idea how did that happend and how I have missed that one...